### DISCUSSION TITLE
Error - undefined method `with_kept_variant_sku_cont'

### DISCUSSION THREAD

**Discussion by gmoustakas** [solidusio/solidus#4192] [Troubleshooting] [answered]
Hi all, i'm new to Solidus and i'm trying to setup solidus with custom Front-end & Back-end, i've copied all the assets to my app so i can override them, until now everything works perfect and i'm making progress and learning about Solidus at the same time, but i came across this error while i was trying to access the `http://localhost:3000/admin/products`

`undefined method 'with_kept_variant_sku_cont' for Ransack::Search<class: Spree::Product, base: Grouping <combinator: and>>:Ransack::Search`

This error only appears in this case, other than that everything works perfect.

**Comment by jarednorman** [comment, marked as answer]
Hey @gemgr, welcome to the community! Do you mind providing the versions of Solidus, Rails, and Ransack that you're using?

**Reply by gmoustakas** [reply]
Hi, yes of course I can share them. 


![Screenshot from 2021-10-18 21-46-33.png](https://user-images.githubusercontent.com/10109147/137790158-6af9dd49-4009-4121-919c-3713ddf362d7.png)

**Reply by jarednorman** [reply]
Why are you running `ransack` from GitHub, rather than the latest release?

**Reply by gmoustakas** [reply]
Hi, sorry for the late replay, i was on a little holiday, i've never worked before with these gems so i found that in the internet in a Stack overflow thread, so i followed that.

**Reply by jarednorman** [reply]
Do you have the same issue if you use the current release of ransack?

**Comment by waiting-for-dev** [comment]
`#with_kept_variant_sku_cont` was introduced as a ransack scope in 4ae093a and added to the admin interface in 1952e824. Both commits seem to be only in master. I guess that you're using `solidus-backend` from master, but a released version of `solidus-core`.
